app/internal/page/share_code.go

```diff
@@ -75,41 +77,56 @@ func (s *ShareCodeSender) SendCertificateProvider(ctx context.Context, template
 
 func (s *ShareCodeSender) SendAttorneys(ctx context.Context, template notify.TemplateId, appData AppData, lpa *Lpa) error {
 	for _, attorney := range lpa.Attorneys {
-		if attorney.Email == "" {
-			continue
```

Comment:
https://github.com/ministryofjustice/opg-modernising-lpa/pull/448/files#diff-f333e5743cd9bcf25c4a96d79874a1aa43b4ab3e172bc9eafac37b9df5c04aeeR95 it's still logically the same, I just moved it into the function instead of checking in the loop body

Reply:
Ah missed that - cool all good then 👍